jsonnet/components/cluster-monitoring-operator.libsonnet

```
@@ -409,6 +409,25 @@ function(params) {
     ],
   },
 
+  // This role enables read access to the platform Alertmanager API
+  // through kube-rbac-proxy.
+  monitoringAlertmanagerViewRole: {
+    apiVersion: 'rbac.authorization.k8s.io/v1',
+    kind: 'Role',
+    metadata: {
+      name: 'monitoring-alertmanager-view',
```

To make it consistent with the `monitoring-alertmanager-api-reader` role in openshift-user-workload-monitoring

```suggestion
      name: 'monitoring-alertmanager-api-reader',
```

---

ACK, in that case, do we want to rename `monitoring-alertmanager-edit` to `monitoring-alertmanager-api-writer` as well, just so `monitoring-alertmanager-edit` and `monitoring-alertmanager-api-reader` don't cause ambiguity?

---

we could create a `monitoring-alertmanager-api-writer` role but we shouldn't remove any role which is already documented (at least not without a deprecation period and ability to tell users that they need to migrate).
But on second thought, yes it might be better to stick with consistency and keep `monitoring-alertmanager-view` for now.

---

Right, makes sense.